includes/validation/class-amp-validation-manager.php

```diff
@@ -526,6 +526,7 @@ public static function init_validate_request() {
 	public static function add_validation_error_sourcing() {
 		add_action( 'wp', [ __CLASS__, 'wrap_widget_callbacks' ] );
 
+		add_filter( 'register_block_type_args', [ __CLASS__, 'wrap_block_callbacks' ], 9 );
```

Comment:
It seems this should perhaps go as early as possible:

```suggestion
		add_filter( 'register_block_type_args', [ __CLASS__, 'wrap_block_callbacks' ], PHP_INT_MIN );
```